docs/products/kafka/howto/kcat.rst

```diff
@@ -97,9 +98,9 @@ Use the following code to consume messages coming from a topic named ``test-topi
 * ``-C``: sets for consumer mode
 * ``-t``: specifies the topic again 
 * ``-o``: defines the topic starting message offset (negative values are considered relative to the latest offset)
-* ``-e``: stops kcat once the end of the topic is reached; without it, it will continuously poll Apache Kafka for new messages.
+* ``-e``: stops `kcat` once the end of the topic is reached; without it, it will continuously poll Apache Kafka for new messages.
```

Comment:
```suggestion
* ``-e``: stops ``kcat`` once the end of the topic is reached; without it, it will continuously poll Apache Kafka for new messages.
```

Reply:
👍